warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java

```diff
@@ -152,11 +158,33 @@ protected void redistributeQueries(Multimap<String,QueryPlan> serverPlan, Tablet
                     rangeIter = Iterables.concat(rangeIter, rangeEntry.getValue());
                 }
 
-                if (log.isTraceEnabled())
+                if (log.isTraceEnabled()) {
                     log.trace("Adding query tree " + JexlStringBuildingVisitor.buildQuery(currentPlan.getQueryTree()) + " " + currentPlan.getSettings().size()
                                     + " for " + server);
+                }
+
+                // update settings before building new query plan
+                List<IteratorSetting> updatedSettings = new ArrayList<>();
```

Comment:
Before we were simply passing the original settings on.  Why are we reconstituting this here?

Reply:
This is a lift and shift of code that happens inside the constructor `public QueryPlan(JexlNode queryTree, Iterable<Range> rangeIter, List<IteratorSetting> settings, Collection<String> columnFamilies)`.

I didn't particularly care that iterator setting updates were hidden inside a single constructor, so I moved it out to be more explicit about what's happening.